form-editor-cae/pom.xml

```diff
@@ -226,4 +229,19 @@
       </dependency>
     </dependencies>
   </dependencyManagement>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <configuration>
+          <ignoredNonTestScopedDependencies combine.children="append">
+            <!-- these dependency is never used directly, but ContentBean-usages cannot be compiled if the dependency is scoped with "test" -->
+            <nonTestScopedDependency>com.coremedia.cms:cap-unified-api</nonTestScopedDependency>
```

Comment:
Couldn't you simply include the dep as `provided` instead?

Reply:
Yup, that works, see the new commit